lib/components/Input/text-field-with-steppers/index.tsx

```diff
@@ -7,55 +7,60 @@ import {
     LabelPairedMinusSmRegularIcon,
 } from "@deriv/quill-icons";
 import "./text-field-with-steppers.scss";
-import { TLeftOrRight } from "@types";
 
 export interface TextFieldWithSteppersProps extends ComponentProps<typeof Input> {
-    unit?: string;
     decimals?: number;
-    unitPlacement?: TLeftOrRight;
+    unitLeft?: string;
+    unitRight?: string;
 }
 
 export const TextFieldWithSteppers = forwardRef<HTMLInputElement, TextFieldWithSteppersProps>(
     (props, ref) => {
-        const { value, onChange, disabled, textAlignment, placeholder = 0, unit = '', decimals = 2, unitPlacement = 'left' } = props;
+        const { value, onChange, disabled, textAlignment, placeholder = 0, unitLeft = '', unitRight = '', decimals = 2, ...rest } = props;
 
         const getFormatValue = (value: number) => parseFloat(value.toFixed(decimals));
 
         const steppersSectionRight = (
             <>
-                {textAlignment !== 'center' && (<button className={clsx("quill-input-steppers-decrement", disabled && 'quill-input-steppers--disabled')}
+                {textAlignment !== 'center' && (<button className={clsx("quill-input-steppers-button", "quill-input-steppers-button--decrement")}
                     onClick={() => {
                         if (!disabled) {
```

Comment:
I'm not sure, bit if button is `disabled` user can't click, so this check might be additional

Reply:
Please, ignore if irrelevant